src/common/GlobalStyles.js

```diff
@@ -4,8 +4,16 @@ export default createGlobalStyle`
   * {
     box-sizing: border-box;
   }
-
+:root {
+  --main-bg-color: #F77D33;
+}
   body {
-    margin: 0;
+    margin: 0 auto;
+    max-width: 1200px;
```

Comment:
Why so big?

Reply:
I thought this would be an appropriate max-width for desktop view, that allows us to keep the mobile design on desktop resolutions. What do you think?

Reply:
I think it is best to make no assumptions about screen size, by default a div will be full width anyway.

Reply:
Sorry, maybe my answer was confusing. Let me add 2 screenshots:
Without max-width (our mobile first design looks too stretched):
![image](https://user-images.githubusercontent.com/59568178/81391492-7880af80-911d-11ea-9198-7b44dfae5f22.png)
With max-width: 
![image](https://user-images.githubusercontent.com/59568178/81391431-5be47780-911d-11ea-9ebb-5eaf34042991.png)

I think max-width helps us to just focus on the mobile design and still have a more or less decent look on wide desktop screens.

Reply:
I see :). Makes sense!